picard/ui/widgets/editablelistview.py

```diff
@@ -209,7 +209,7 @@ def sort(self, column, order=QtCore.Qt.SortOrder.AscendingOrder):
     def get_display_name(self, item):  # pylint: disable=no-self-use
         return item
 
-    def rowCount(self, parent=QtCore.QModelIndex()):
+    def rowCount(self, parent=QtCore.QModelIndex()):  # noqa: B008
```

Comment:
I think this really should be rewritten to `parent=None` and doing a 

```
if parent is None:
    parent=QtCore.QModelIndex()
```

inside the function. I know https://doc.qt.io/qt-6/qabstractitemmodel.html#rowCount shows it that way it is defined here, but that's C++. And there the function providing the default argument does get invoked when the actual function is called (without the argument provided).

Same for `insertRows` and `removeRows` below.